flask_resty/spec/plugin.py

```diff
@@ -30,7 +30,7 @@ def schema_path_helper(spec, path, view, **kwargs):
             'type': 'string',
         })
     if parameters:
-        path['parameters'] = parameters
+        operations['parameters'] = parameters
```

Comment:
This ends up at the same place... but why? Why is `parameters` a key under operations?